libats-http/src/main/scala/com/advancedtelematic/libats/http/HealthResource.scala

```diff
@@ -40,7 +41,8 @@ class HealthResource(versionRepr: Map[String, Any] = Map.empty,
 
         val healthRoutes =
           pathEnd {
-            val f = Future.sequence(healthChecks.map(_ (logger))).map(_ => Map("status" -> "OK"))
+            val f = Future.sequence(healthChecks.map(_ (logger))).map(_ => StatusCodes.OK -> Map("status" -> "OK"))
+              .recover { case _ => StatusCodes.ServiceUnavailable -> Map("status" -> "DOWN")}
```

Comment:
Would be useful to know why? we could just log the exception?

Reply:
`akka-http` will turn it into `500 Internal Server Error`, which is kind of confusing IMO.

Reply:
yes agreed, but I meant either logging the error before returning 503, or returning the exception in the body.

```
recover { case ex =>  
   logger.error("health status error", ex)
   StatusCodes.ServiceUnavailable -> Map("status" -> "DOWN")}
}
```

Reply:
AFAIU we are passing a logger to health check to do it. Or it was not intention?

Reply:
well yes but we cannot really rely on the health check to do it before returning a failed future? Logger in the health check is more so that the health can log additional information?

Reply:
Fair enough. Will add logging and merge